RELEASE_CHANGELOG.md

```diff
@@ -0,0 +1,205 @@
+## [v1.19.0-rc1](https://github.com/provenance-io/provenance/releases/tag/v1.19.0-rc1) - 2024-06-24
+
+### Features
+
+* Bump cosmos-SDK to `v0.50.2` (from `v0.46.13-pio-2`) [#1772](https://github.com/provenance-io/provenance/issues/1772).
```

Comment:
I believe we are on 50.6 now? We can add that context to the overall final release notes though since this specific entry was for this specific older version.